react-native/react/styles/common.js

```diff
@@ -62,7 +60,7 @@ const buttons = {
   }
 }
 
-export default StyleSheet.create({
+export default sheet({
```

Comment:
This whole thing is going away soon and we'll just use style-guide, but for now remove StyleSheet from the desktop side as it never really existed